specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Yes :) That's the way I've had to handle this stuff since 2008 in `electionaudits`.